templates/docker/nginx.conf

```diff
@@ -0,0 +1,16 @@
+worker_processes 1;
+
+events {
+  worker_connections 1024;
+}
+
+stream {
+  upstream ankerctl {
+    server ankerctl:5000;
+  }
+
+  server {
+    listen 4470;
+    proxy_pass ankerctl;
```

Comment:
At this point, we are just letting nginx forward tcp connections. Why is it that we are even using nginx?

In the future, we probably want to run the flask app using uwsgi, and then nginx will be highly useful. But right now I'm not sure I'm seeing the appeal?